src/dataDisplay/Icon/index.tsx

```diff
@@ -124,6 +127,7 @@ const icons = {
   fuelIndicator,
   getInTouch,
   home,
+  importImg,
```

Comment:
Could we rename this to just: "import"?

Reply:
Import and export are reserved words in js. We could think of giving them another name.

Reply:
Sure! oops. Let's leave it as it is.